src/clr-angular/layout/tabs/tab-link.directive.ts

```diff
@@ -29,9 +30,8 @@ let nbTabLinkComponents: number = 0;
     '[attr.aria-hidden]': 'false',
     '[attr.aria-controls]': 'ariaControls',
     '[class.btn]': 'true',
-    '[class.btn-link]': '!inOverflow',
-    '[class.nav-link]': '!inOverflow',
-    '[class.nav-item]': '!inOverflow',
+    '[class.btn-link]': 'tabsService.orientation === "vertical" || !inOverflow',
+    '[class.nav-link]': 'tabsService.orientation === "vertical" || !inOverflow',
```

Comment:
Would you mind adding a test that checks for the correct class on the button when the orientation changes?

Reply:
Will add a test

Reply:
Done. Added tests for overflow too.